_posts/2017-07-21-pydelhi-conf-2017-a-beautiful-conference-happend-at-new-delhi-india.md

```diff
@@ -0,0 +1,235 @@
+[meetups][pydelhi_meetup]. This community has great people who are always happy
+to mentor.
+
+
+### [PyDelhi conf 2017][pydelhi_conf_2017]
+
+
+![Conference T-shirt]({{site.url}}/assets/images/pydelhi_conf_2017/t_shirt.jpg)
+
+<p class="center"> Conference T-shirt </p>
+
+PyDelhi conf is a regional level conference of Python programming language
+organized by PyDelhi community. It is their second year organizing this
+conference. Last year it was located at [JNU University](http://www.jnu.ac.in).  This
+year it happened at [IIM, Lucknow](https://www.iiml.ac.in/) campus based in
+Noida, New Delhi, India. I enjoyed various talks which I will mention later
+here, a workshops section because I was conducting one and some the panel
+discussions because people involved were having a good level of experience. 80%
+of the time slot was divided equally between 30 minutes talk and 2-hour workshop
+section.  10% were given to panel discussions and 10% was preserved for
+lightning talks.  The dev sprints were happening parallel with the conference.
+The early slot was given to workshops for both the days. One large conference
+hall was located on a 2nd floor of the building and two halls were at a ground
+floor.  Food and beverages were served nicely on the base floor.
+
+![Panel discussion]({{site.url}}/assets/images/pydelhi_conf_2017/pannel_disussion.jpg)
+
+<p class="center"> Panel Discussion </p>
+
+![Desk]({{site.url}}/assets/images/pydelhi_conf_2017/desk.jpg)
+
+<p class="center"> Registration desk </p>
+
+![Lunch]({{site.url}}/assets/images/pydelhi_conf_2017/lunch.jpg)
+
+<p class="center"> Tea break </p>
+
+
+### Keynote speakers
+
+
+![Mr.Richardo Rocha]({{site.url}}/assets/images/pydelhi_conf_2017/ricardo.jpg)
+
+* [**Mr.Ricardo Rocha:**](
+  https://www.linkedin.com/in/ricardo-rocha-739aa718/?ppe=1) Mr.Rocha is a
+  software engineer at [CERN][cern]. I got some time to talk with him
+  post-conference. We discussed what are his responsibilities at [CERN][cern]. I
+  was impressed when he explained how he is managing infrastructure with his
+  team. On inquiring opportunities available at [CERN][cern] he mentioned that
+  the organization is always looking for the talented developers.  New grads can
+  keep an eye on various Summer Internship Programs which are very similar to
+  Google Summer of Code program.
+
+![Mr.Chris Stucchio]({{site.url}}/assets/images/pydelhi_conf_2017/chris.jpg)
+
+* [**Mr.Chris Stucchio:**](https://www.chrisstucchio.com/) Mr.Stucchio is
+  director of Data Science at [Wingify/ VWO](https://vwo.com/). I found him
+  physically fit compared to other software developers (mostly of India). I
+  didn’t get much time to have a word with him. Keynote speech provided by him
+  was very much impressive.
+
+
+### Interesting Talks
+
+Because I took the wrong metro train, I was late for the inaugural ceremony. I
+also missed a keynote given by Mr.Rocha. Below talks were impressively
```

Comment:
In the section above, you mention that the speech was "very much impressive". Here you say you missed it. Very much confusing.